scripts/ci/perf_test.sh

```diff
@@ -44,7 +44,7 @@ done
 trap "aws --region us-west-2 ec2 terminate-instances --instance-ids $ec2_instance_id" KILL
 
 # push binary and script to instance
-debname=$(find /tmp/workspace/artifacts/influxdb2*amd64.deb)
+debname=$(find /tmp/workspace/artifacts -regex ".*/influxdb2-[0-9].*amd64.deb")
```

Comment:
This is the one difference from `master`. Since we're still building the `influxdb2-client*.deb` in this branch we need to be more specific with the `find` to make sure we only get 1 file.